myriad-scheduler/src/main/java/com/ebay/myriad/scheduler/fgs/OfferUtils.java

```diff
@@ -0,0 +1,35 @@
+package com.ebay.myriad.scheduler.fgs;
+
+import java.util.Collection;
+import org.apache.hadoop.yarn.api.records.Resource;
+import org.apache.mesos.Protos;
+import org.apache.mesos.Protos.Offer;
+
+/**
+ * Utility class that provides useful methods that deal with Mesos offers.
+ */
+public class OfferUtils {
+
+  /**
+   * Transforms a collection of mesos offers into {@link Resource}.
+   *
+   * @param offers collection of mesos offers
+   * @return a single resource object equivalent to the cumulative sum of mesos offers
+   */
+  public static Resource getYarnResourcesFromMesosOffers(Collection<Offer> offers) {
+    double cpus = 0.0;
+    double mem = 0.0;
+
```

Comment:
I'd argue there's a few use cases, especially for machines with faster but fewer cores and lots of RAM.  I could also see the ratio being greater than one as in Mesos you can increase cpu resource past the physical cores of the machine (I did this so I could test FGS).